### PR TITLE
Sendability fixes

### PR DESCRIPTION
**Pull request by michalbencur**
Fixing small issues with Swift Concurrency.

### CODE REVIEW

**Comment by Joannis** [comment]
Thanks for the PR! I've tagged myself as reviewer - otherwise I miss these PRs

File: Sources/Citadel/SFTP/SFTPFileFlags.swift [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import Foundation
 
 // pflags
-public struct SFTPOpenFileFlags: OptionSet, CustomDebugStringConvertible {
-    public var rawValue: UInt32
+public struct SFTPOpenFileFlags: OptionSet, CustomDebugStringConvertible, Sendable {
+    public let rawValue: UInt32
```
Comment (Joannis) [inline Sources/Citadel/SFTP/SFTPFileFlags.swift:5]: This is a breaking change

Comment (Joannis) [reply Sources/Citadel/SFTP/SFTPFileFlags.swift:5]:
```suggestion
    public var rawValue: UInt32
```

**Comment by michalbencur** [comment]
> Thanks for the PR! I've tagged myself as reviewer - otherwise I miss these PRs

Thank you for review, I've pushed your suggestion.

**Review by Joannis — APPROVED** [review]

**Pull request merged**